Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7247,6 +7265,2013 @@ def panorama_install_file_content_update_command(args: dict):
+                summary_data.append(ShowRoutingRouteSummaryData(
+                    hostid=firewall_hostname,
+                    interface=interface,
+                    route_count=route_count
+                ))
+
+        return ShowRoutingRouteCommandResult(
+            summary_data=summary_data,
+            result_data=result_data
+        )
+
+
+"""
+-- XSOAR Specific Code Starts below --
+"""
+
+
+def test_topology_connectivity(topology: Topology):
+    """To get to the test-module command we must connect to the devices, thus no further test is required."""
+    if len(topology.firewall_objects) + len(topology.panorama_objects) == 0:
+        raise ConnectionError("No firewalls or panorama instances could be connected.")
+
+    return "ok"
+
+
+def get_arp_tables(topology: Topology, device_filter_string: str = None) -> ShowArpCommandResult:
+    """
+    Gets all arp tables from all firewalls in the topology.
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    result: ShowArpCommandResult = FirewallCommand.get_arp_table(topology, device_filter_string)
+    return result
+
+
+def get_route_summaries(topology: Topology,
+                        device_filter_string: str = None) -> ShowRouteSummaryCommandResult:
+    """
+    Pulls all route summary information from the topology
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    result: ShowRouteSummaryCommandResult = FirewallCommand.get_routing_summary(topology,
+                                                                                device_filter_string)
+    return result
+
+
+def get_routes(topology: Topology,
+               device_filter_string: str = None) -> ShowRoutingRouteCommandResult:
+    """
+    Pulls all route summary information from the topology
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    result: ShowRoutingRouteCommandResult = FirewallCommand.get_routes(topology,
+                                                                       device_filter_string)
+    return result
+
+
+def get_system_info(topology: Topology,
+                    device_filter_string: str = None) -> ShowSystemInfoCommandResult:
+    """
+    Gets information from all PAN-OS systems in the topology.
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    result: ShowSystemInfoCommandResult = UniversalCommand.get_system_info(topology,
+                                                                           device_filter_string)
+    return result
```

Comment:
```suggestion
    return UniversalCommand.get_system_info(topology, device_filter_string)
```

Reply:
You can do the same for all the functions below as well

Reply:
Done